wifiphisher/pywifiphisher.py

```diff
@@ -106,6 +106,11 @@ def parse_args():
         "--presharedkey",
         help=("Add WPA/WPA2 protection on the rogue Access Point. " + 
               "Example: -pK s3cr3tp4ssw0rd"))
+    parser.add_argument(
+        "-qS",
+        "--quitonsuccess",
+        help="Stop Wifiphisher after successful phishing attack.",
```

Comment:
I would rephrase this to "Stop the script after successfully retrieving one pair of credentials."